_inc/client/components/product-expiration/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { moment, translate as __, localize } from 'i18n-calypso';
+
+class ProductExpiration extends React.PureComponent {
+	static propTypes = {
+		expiry_date: PropTypes.string,
+		subscribed_date: PropTypes.string,
+		is_refundable: PropTypes.bool,
+	};
+
+	static defaultProps = {
+		expiry_date: '',
+		subscribed_date: '',
+		is_refundable: false,
+	};
+
+	render() {
+		const { expiry_date, subscribed_date, is_refundable } = this.props;
+
+		// Return nothing if we don't have any dates.
+		if ( ! expiry_date && ! subscribed_date ) {
+			return null;
+		}
+
+		// Return the subscription date if we don't have the expiry date or the plan is refundable.
+		if ( ! expiry_date || is_refundable ) {
+			return __( 'Purchased on %s.', { args: moment( subscribed_date ).format( 'LL' ) } );
```

Comment:
Not sure what you mean here. As in, allow the format to be passed in as a prop?

Reply:
Not as a prop, but defined somewhere in a var that we use in all cases instead of specifying `'LL'` every time. Just so if we want to change it, we'll change it once instead of 3 times.

Reply:
Ahh, the `LL` part! I thought you meant the `Purchased on %s` part. Yes, that makes sense.

Reply:
Done with 42e250c2a527d9eb7ad4b9c1140caa3106dcbaa4